test/manual/amp-sidebar-1.0.amp.html

```diff
@@ -209,7 +209,7 @@
     </ul>
 
     <ul class="ampstart-sidebar-faq list-reset m0">
-      <li toolbar-only class="ampstart-faq-item"><a href="#" class="text-decoration-none">About</a></li>
+      <li hidden-when-toolbar class="ampstart-faq-item"><a href="#" class="text-decoration-none">About</a></li>
```

Comment:
Grrr. I'm still reading this as "hidden in the toolbar", as in show only in sidebar. Maybe that's how we should approach it? `show-in="toolbar|sidebar|toolbar,sidebar"`? With default, both "toolbar,sidebar"?